lib/features/manage_account/presentation/model/account_menu_item.dart

```diff
@@ -71,6 +76,8 @@ enum AccountMenuItem {
         return 'vacation';
       case AccountMenuItem.mailboxVisibility:
         return 'folder-visibility';
+      case AccountMenuItem.notification:
+        return 'notification';
```

Comment:
we need it for browser? can you have demo also in web responsive?

Reply:
Demo added